oauth2_provider/admin.py

```diff
@@ -38,11 +44,15 @@ class GrantAdmin(admin.ModelAdmin):
 class IDTokenAdmin(admin.ModelAdmin):
     list_display = ("jti", "user", "application", "expires")
     raw_id_fields = ("user",)
+    search_fields = ("token",) + (("user__email",) if has_email else ())
```

Comment:
Is this valid? Searching from the admin now gives me `django.core.exceptions.FieldError: Cannot resolve keyword 'token' into field. Choices are: access_token, application, application_id, created, expires, id, jti, scope, updated, user, user_id`